### PR TITLE
Turn on LLL testing in test_ubuntu1904_clang

### PR DESCRIPTION
**Pull request by axic**
Depends on #7853

### CODE REVIEW

File: test/liblll/Compiler.cpp [hunk 1]
```diff
@@ -193,6 +193,8 @@ BOOST_AUTO_TEST_CASE(valid_opcodes_functional)
 		"4300",
 		"4400",
 		"4500",
+		"4600",
```
Comment (ekpyron) [inline test/liblll/Compiler.cpp]: Ah, and in this test they *were* added to the other vector - that exactly explains the weird failures we had on the osx build earlier :-).

**Review by ekpyron — DISMISSED** [review]
Why the clang one in particular? We could enable it everywhere for the time being, if we remove it again soonish anyways - and as long as we keep it we should probably test it on all platforms to be safe...

File: test/liblll/Compiler.cpp (hunk 1 above)
Comment (axic) [reply test/liblll/Compiler.cpp]: Yeah. It was shifted. (Left the note in gitter.)

**Comment by axic** [comment]
> Why the clang one in particular? We could enable it everywhere for the time being, if we remove it again soonish anyways - and as long as we keep it we should probably test it on all platforms to be safe...

I don't mind. Can turn on by default, it shouldn't matter too much speed wise.

**Comment by axic** [comment]
@ekpyron do you want to take this over given you know the circleci config better?

**Comment by ekpyron** [comment]
Labelling this as "to discuss" for Monday to decide for which CI runs to enable it - once decided I can take it over and do it, sure!

**Comment by axic** [comment]
Agreed on today's meeting to just enabled it for all soltest runs.

@ekpyron also make sure to rebase this as the actual fix was merged by #7853.

**Comment by axic** [comment]
@ekpyron need to disable for the asan task given it leaks memory (it is a known issue, see #2932)

**Comment by ekpyron** [comment]
@axic Ah, right - I was curious to see, if it finds errors, but didn't know that it's a known issue - I'm disabling.

**Review by ekpyron — DISMISSED** [review]
Note that my review on this one is not really worth much, since I basically rewrote it.

**Comment by ekpyron** [comment]
Note that the LLL tests occur in the XML artifacts of the CI runs, so they are in fact run as intended.

**Comment by axic** [comment]
> Note that the LLL tests occur in the XML artifacts of the CI runs, so they are in fact run as intended.

@ekpyron Confirmed in `t_osx_soltest` and `t_ubu_soltest`, but not in `t_ubu_release_soltest`.

**Comment by ekpyron** [comment]
Weird.. I admit I didn't check that one - apparently it doesn't import the environment variable from the normal build - I'm enabling it explicitly for that one as well - this time it should be there.

**Comment by ekpyron** [comment]
Now I see it in the ``t_ubu_release_soltest`` artifact as well.

**Comment by axic** [comment]
This works now!

**Comment by axic** [comment]
Oh well, should have updated the commit message.

**Comment by ekpyron** [comment]
> Oh well, should have updated the commit message.

True...

**Pull request merged**